admin-ui/src/test/java/io.crate/rest.action.admin/AdminUIIntegrationTest.java

```diff
@@ -56,6 +56,27 @@ public void testLegacyRedirect() throws IOException, URISyntaxException {
     }
 
     @Test
+    public void testPluginURLRedirect() throws IOException, URISyntaxException {
+        //request to '/_plugin/crate-admin' is redirected to '/index.html'
+        Header[] headers = {
+            browserHeader()
+        };
+
+        List<URI> allRedirectLocations = getAllRedirectLocations("/_plugin/crate-admin", headers);
+
+        // all redirect locations should not be null
+        assertThat(allRedirectLocations, notNullValue());
+        // all redirect locations should contain the crateAdminUI URI
+        assertThat(allRedirectLocations.contains(adminURI()), is(true));
+    }
+
+    @Test
+    public void testPluginURLRedirectReturnsIndex() throws IOException, URISyntaxException {
+        //request to '/_plugins/crate-admin' is redirected to '/index.html'
+        assertIsIndexResponse(browserGet("/_plugin/crate-admin"));
```

Comment:
/_plugins/crate-admin => redirects to / => responds with index.html

@seut the first test checks if /_plugins/crate-admin redirects to /. 
The second tests if requesting /_plugins/crate-admin responds with  index.html